InternetShop/src/main/java/dao/OrderDaoImpl.java

```diff
@@ -0,0 +1,38 @@
+package dao;
+
+import model.Order;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class OrderDaoImpl implements OrderDao {
+    private static Map<Long, Order> orders = new HashMap<>();
+
+    @Override
+    public List<Order> getAllOrders() {
+        return new ArrayList(orders.values());
+    }
+
+    @Override
+    public void createOrderInDatabase(Order order) {
+        orders.put(order.getId(), order);
```

Comment:
need to check that we have null check for Long id field on service layer, or add it here